unlock-app/src/middlewares/storageMiddleware.js

```diff
@@ -64,6 +59,8 @@ const storageMiddleware = config => {
 
     // SIGNUP_CREDENTIALS
     storageService.on(success.createUser, publicKey => {
+      // TODO: Dispatch a gotEncryptedPrivateKeyPayload instead of
+      // setting here, will need to change what storageService emits
       dispatch(setAccount({ address: publicKey }))
```

Comment:
Updating this to use `gotEncryptedPrivateKeyPayload` is necessary for full functionality, but I think should be deferred to a future PR for the sake of making this one easily reviewable